bcl2fastq/index_summary.py

```diff
@@ -44,18 +42,27 @@
                 -  multiple select gives error due to inconsistent values
           """
 layout = html.Div(children=[
+    dcc.ConfirmDialog(
+        id='error',
+        message='You have input an incorrect run. Click either "Ok" or "Cancel" to return to the most recent run.'
```

Comment:
Since clicking either button brings the user back to the same place, it would make more sense for the user if we use an "OK" dialog instead of a "Confirm" dialog (with updated message text to match).